load.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * SEO Module
+ *
+ * @package hm-platform/seo
+ */
+
+namespace HM\Platform;
+
+require_once __DIR__ . '/inc/namespace.php';
+
+add_action( 'hm-platform.modules.init', function () {
+	register_module( 'seo', __DIR__, 'SEO', [
+		'enabled' => false,
```

Comment:
I feel like maybe we can default this on? We are defaulting off the more controversial stuff anyway